scripts/gen_ldelf_hex.py

```diff
@@ -9,9 +9,24 @@
 
 import argparse
 import sys
-from elftools.elf.elffile import ELFFile
-from elftools.elf.sections import SymbolTableSection
-from elftools.elf.constants import P_FLAGS
+try:
+    from elftools.elf.elffile import ELFFile
+    from elftools.elf.sections import SymbolTableSection
+    from elftools.elf.constants import P_FLAGS
+except ImportError:
+    print("""
+***
+Can't find elftools module. Probably it is not installed on your system.
+You can install this module with
+
+# apt install python-pyelftools
```

Comment:
That should be `python3-pyelftools`. Also `$ sudo apt install ...` instead of `# apt install ...` maybe?